aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -16,6 +16,8 @@
 #include <ATen/Parallel.h>
 #include <ATen/native/UnaryOps.h>
 #include <ATen/native/TensorIterator.h>
+#include <ATen/cpu/vec256/vec256_base.h>
```

Comment:
Why is this one needed?

Reply:
This is old from the original implementation. Can delete